src/test/kotlin/no/nav/familie/ef/sak/config/PdlClientConfig.kt

```diff
@@ -4,21 +4,21 @@ import io.mockk.every
 import io.mockk.mockk
 import no.nav.familie.ef.sak.integration.PdlClient
 import no.nav.familie.ef.sak.integration.dto.pdl.*
-import no.nav.familie.util.FnrGenerator
 import org.springframework.context.annotation.Bean
 import org.springframework.context.annotation.Configuration
 import org.springframework.context.annotation.Primary
 import org.springframework.context.annotation.Profile
 import java.time.LocalDate
 import java.time.LocalDateTime
+import java.time.Month
 
 @Configuration
 @Profile("mock-pdl")
 class PdlClientConfig {
 
     private val startdato = LocalDate.of(2020, 1, 1)
     private val sluttdato = LocalDate.of(2021, 1, 1)
-    private val barnFnr = FnrGenerator.generer(2020, 1, 1)
+    private val barnFnr = "02022088888"
```

Comment:
Er dette et gyldig fødselsnummer? Nice i så fall 😄

Reply:
Tror ikke det, dessverre... Men `01010172272` er gyldig :)

Reply:
For de som lurer, det var ikke gyldig 😢